cwl/src/test/scala/wdl4s/cwl/ExportCwlSamplesSpec.scala

```diff
@@ -0,0 +1,64 @@
+package wdl4s.cwl
+
+import io.circe.generic.auto._
+import org.scalatest.{FlatSpec, Matchers}
+import io.circe.shapes._ // required, even though IntelliJ thinks it is unused
+import io.circe.syntax._
+import io.circe.refined._
+import io.circe.literal._
+import wdl4s.cwl.CwlEncoders._
+import shapeless.Coproduct
+
+/**
+  * wdl4s
+  * Created by oruebenacker on 03.07.17.
+  */
+class ExportCwlSamplesSpec extends FlatSpec with Matchers {
+
+  it should "export 1st tool" in {
+    val tool =
+      CommandLineTool(
+        inputs = Coproduct[CommandLineTool.Inputs](Map("message" -> CommandInputParameter(
+          id = None,
+          label = None,
+          secondaryFiles = None,
+          format = None,
+          streamable = None,
+          doc = None,
+          inputBinding = Some(CommandLineBinding(
+            loadContents = None,
+            position = Some(1),
```

Comment:
Ok, I switched my `Some`s to `Option`s. I had to hold my hand to keep it from switching them back.

Out of curiosity: Do you mean consistency is simply much more important than the small performance loss, or do you expect the compiler to optimize away the extra null check?